Makefile

```diff
@@ -264,7 +269,7 @@ e2e-coverage:
 .PHONY: kind-load
 kind-load: $(KIND) #EXHELP Loads the currently constructed images into the KIND cluster.
 	$(CONTAINER_RUNTIME) save $(IMG) | $(KIND) load image-archive /dev/stdin --name $(KIND_CLUSTER_NAME)
```

Comment:
`IMG` should be renamed to `OPCON_IMG` or `OPERATOR_CONTROLLER_IMG` for symmetry.

Reply:
It is possible to introduce the `IMG_NAMESPACE` variable however, it seems that the `OPERATOR_CONTROLLER_IMAGE_REPO` and `CATALOGD_IMAGE_REPO` variables can not be removed as they would be passed to the release target in the make file.

Reply:
done. Let me know if the current  solution works